srcpkgs/gnome-boxes/template

```diff
@@ -9,12 +9,12 @@ makedepends="clutter-gtk-devel freerdp-devel gtk-vnc-devel libarchive-devel
  libglib-devel libgudev-devel libosinfo-devel libsecret-devel libsoup-devel
  libusb-devel libvirt-glib-devel libxml2-devel spice-gtk-devel spice-protocol
  tracker-devel vala-devel webkit2gtk-devel vte3-devel gtksourceview4-devel
```

Comment:
`tracker-devel` here is not needed, `gnome-boxes` has been ported to `tracker3`

Reply:
fixed